frontend/src/styles/normalize.css

```diff
@@ -0,0 +1,131 @@
+/*! normalize.css v8.0.0 | MIT License | github.com/necolas/normalize.css */
+
+/* Document
+   ========================================================================== */
```

Comment:
Looks like this is the normalize.css file included, but in the ticket we were hoping to indtroduce Sass as well which is missing in this PR. So what we would be hoping to have is:
-  a `src/styles` folder with a `styles.scss`  that uses sass and has something like `@import 'normalize';` (the file you have now is perfectly in place for this import)
-  a dist folder that once sass is compiled in the command line has the compiled css result which we also want to commit for now in this PR.

Please re-review the section TODO in the ticket for more details https://trello.com/c/HMHV8gVI/32-sass-setup-normalizecss-kit

Does it make sense? Please do say if you want more details/explanation more than happy to jump on a video chat or slack to discuss further.